samples/client/echo_api/python/openapi_client/models/pet.py

```diff
@@ -20,20 +20,21 @@
 
 
 from typing import List, Optional
-from pydantic import BaseModel, Field, StrictInt, StrictStr, conlist, validator
+from pydantic import BaseModel, StrictInt, StrictStr, validator
 from openapi_client.models.category import Category
 from openapi_client.models.tag import Tag
+from pydantic import Field
 
 class Pet(BaseModel):
     """
     Pet
     """
     id: Optional[StrictInt] = None
-    name: StrictStr = Field(...)
+    name: StrictStr
```

Comment:
`= Field(...)` was added on purpose, we should not remove it. See https://github.com/OpenAPITools/openapi-generator/pull/15290

Reply:
I think passing `...` to `Field()` is required for all fields that can be `None`, but are required (`None` is not the default value). Also in request function signatures.

Reply:
I will check how to add a test for this, if there's not one yet already :+1:

Reply:
@robertschweizer  IIRC, this is only a typing issue, not a runtime issue, right?

If yes, I'd propose to let this aside from the time being and mark this as "TODO to improve typing support for the Pydantic v2-compatible generator". Even with all the changes I did here, there's still a long way for a proper typing support, and I guess it will be part of it.

I'll note your comment somewhere though so to not forget it in the new Pydantic v2 generator.

Reply:
@robertschweizer Correct me if I'm wrong, but I think the new format should be fine with Pydantic v2:

```python
from typing import Optional

from pydantic import BaseModel, StrictStr


class Model(BaseModel):
    required_field: StrictStr
    required_nullable_field: Optional[StrictStr] = None


m1 = Model(required_field="foo", required_nullable_field="bar")

print(m1.required_field.lower())
if m1.required_nullable_field is not None:
    print(m1.required_nullable_field.lower())


m2 = Model(required_field="m2")

print(m2.required_field.lower())
if m2.required_nullable_field is not None:
    print(m2.required_nullable_field.lower())
```
prints:

```
m1
m1 nullable
m2
```

`mypy --strict` says:
```
Success: no issues found in 1 source file
```

pylint says nothing too (only missing docstrings).

If you have another specific example in mind, could you share it and I can take a look!